lib/maintainers.nix

```diff
@@ -279,6 +279,7 @@
   hodapp = "Chris Hodapp <hodapp87@gmail.com>";
   hrdinka = "Christoph Hrdinka <c.nix@hrdinka.at>";
   htr = "Hugo Tavares Reis <hugo@linux.com>";
+  hyper_ch = "Stephan Jau <nixos@sjau.ch>";
```

Comment:
I know hyper_ch is your irc handle but this should be your github username so its easier to ping you

Reply:
This will not matter with the merge of #31739.